modules/workflow/src/main/java/org/eclipse/xpanse/modules/workflow/handle/migrate/DestroyFailedManualHandlerListener.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.workflow.handle.migrate;
+
+import lombok.extern.slf4j.Slf4j;
+import org.activiti.engine.RuntimeService;
+import org.activiti.engine.delegate.DelegateTask;
+import org.activiti.engine.delegate.TaskListener;
+import org.eclipse.xpanse.modules.workflow.consts.MigrateConstants;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+/**
+ * Monitoring class for manual processing after destroy failure and retry.
+ */
+@Slf4j
+@Component
+public class DestroyFailedManualHandlerListener implements TaskListener {
+
+    private static RuntimeService runtimeService;
```

Comment:
is there a specific reason for declaring all spring autowired components as static? if not, we must keep it only private and final. Static is not needed. 
This applies to all beans in the workflow module.

Reply:
Yes, it needs to be static. If it is not static, it will report an error.